apps/docs/content/docs/collaboration.mdx

```diff
@@ -0,0 +1,133 @@
+
+```bash
+npm install @tldraw/sync
+```
+
+Back in your code, import the `useSyncDemo` hook from the `@tldraw/sync` library. In the `MyApp` component, call the `useSyncDemo` hook with a room ID. Pass the [`store`](/docs/persistence#The-store-prop) returned by `useSyncDemo` to the tldraw component's `store` prop.
+
+```tsx
+import { Tldraw } from 'tldraw'
+import { useSyncDemo } from '@tldraw/sync'
+
+function MyApp() {
+	const store = useSyncDemo({ roomId: 'my-unique-room-id' })
+	return <Tldraw store={store} />
+}
+```
+
+In your browser, open an incognito window and visit your project's URL. You can also use a different browser or different device. The key is the room ID: any apps connecting to the same room ID will enter into a shared collaboration session.
+
+### Limitations of the demo
+
+The sync demo is great for prototyping but you should not use it in production. Data on the demo server only lasts for up to 24 hours, and anyone using the same room id will be able to edit the same room's project.
+
+### Using tlsync in production
+
+To use tldraw sync in production, you will need to self-host the tldraw sync server. To learn more, see our article on [tldraw sync](/sync).
+
+> We don't offer a hosted solution for tldraw sync in production, but if you're interested in that please let us know at hello@tldraw.com.
+
+## Using other collaboration backends
+
+While tldraw sync is our recommended and best-supported backend for tldraw, we designed the tldraw SDK to work with any data solution.
+
+As a reference, our [tldraw-yjs example](https://github.com/tldraw/tldraw-yjs-example) shows an integration between tldraw and the [Yjs](https://yjs.dev) CRDT library.
+
+Whichever backend you choose, adding collaboration to tldraw will involve sychronizing data, sharing user presence, and handling user assets.
+
+### Synchronizing data
+
+For more information about how to synchronize the tldraw editor's store with other processes (i.e. how to get data out and put data in, including from remote sources) see the [Persistence](/docs/persistence) page.
+
+### User assets
+
+As well as synchronizing the rapidly-changing document data, tldraw also needs a way to store and retrieve large binary assets like images or videos. You'll need to make sure your backend can handle asset uploads and downloads, and implement a service for turning files and links into assets that the editor can use.
+
+You can use the [Editor](?)'s asynchronous external asset handlers to generate assets from content that a user drops, pastes, or adds to the canvas. For example, you might upload images to a storage bucket and return an asset that includes their source URL; or pull favicons and social images in order to create bookmarks from pasted URLs.
+
+```tsx
+<Tldraw
+	store={store}
+	onMount={(editor) => {
+		editor.registerExternalAssetHandler('file', createAssetFromFile)
+		editor.registerExternalAssetHandler('url', createAssetFromUrl)
+	}}
+/>
+```
```

Comment:
This is no longer really accurate. We should remove it or update it to match some of the stuff that was deleted from sync.mdx.

For files, overriding the external asset handler is now rarely needed - it's basically only if you want to customize how we figure out what sort of file it is and how we extract metadata. It doesn't have anything to do with collaboration.

For collaboration, they should have supplied a `TLAssetStore` with an `upload` method when they created their store. That method is how you change where assets code.

As for URLs, I don't see what this has to do with collaboration at all? It's also not really something you can customize per se - you just sort of have to implement it. I don't think it's worth calling out here, this makes it seem like too important a concept